lib/commands/location.js

```diff
@@ -4,6 +4,43 @@ import log from '../logger';
 
 let commands = {};
 
+/**
+ * @typedef {Object} LocationObject
+ *
+ * @property {number} latitude - The latitude of the device under test
+ * @property {number} longitude - The lognitude of the device under test
+ * @property {number} altitude - Always 0 since iOS does not return altitude as the Location Service
```

Comment:
I assume we could also retrieve the altitude value from WDA: https://developer.apple.com/documentation/corelocation/cllocation/1423820-altitude?language=objc

Reply:
ah, nice. I found it in `locationManager.location`. I ignored this since `locationManager.location.coordinate` did not have it. Let me include this in https://github.com/appium/WebDriverAgent/pull/452